Compilation failure in fresh Ubuntu 16.04

Hey, so I'd love to check out the work you guys are doing, but I can't get vg to compile. Normally I'd chalk it up to my configuration or lack of skill with make/gcc, but it also failed on a fresh install of Ubuntu.

I actually set up a fresh VM of Ubuntu 16.04.1 (32-bit) and followed the instructions in the README to the letter.

I installed the VM and got it fully updated, then I installed git and followed the README:
```
$ git clone --recursive https://github.com/vgteam/vg.git
$ cd vg
$ make get-deps
$ source source_me.sh
$ make static
```

But after 15 minutes or so it failed, with the errors starting around this bit:
```
. ./source_me.sh && cd deps/ssw/src && make && ar rs /home/me/src/vg/lib/libssw.a ssw.o ssw_cpp.o && cp ssw_cpp.h ssw.h /home/me/src/vg/lib
make[1]: Entering directory '/home/me/src/vg/deps/ssw/src'
gcc -c -o ssw.o ssw.c -Wall -pipe -O3
ssw.c: In function ‘sw_sse2_byte’:
ssw.c:155:10: warning: SSE vector return without SSE enabled changes the ABI [-Wpsabi]
  __m128i vZero = _mm_set1_epi32(0);
          ^
```
And then there were a few dozen versions of the same error:
```
In file included from ssw.c:38:0:
/usr/lib/gcc/i686-linux-gnu/5/include/emmintrin.h:1180:1: error: inlining failed in call to always_inline ‘_mm_srli_si128’: target specific option mismatch
_mm_srli_si128 (__m128i __A, const int __N)
^
ssw.c:137:29: error: called from here
#define max16(m, vm) (vm) = _mm_max_epu8((vm), _mm_srli_si128((vm), 8)); \
                             ^
ssw.c:292:3: note: in expansion of macro ‘max16’
   max16(maxColumn[i], vMaxColumn);
   ^
```
Finishing up with:
```
Makefile:47: recipe for target 'ssw.o' failed
make[1]: *** [ssw.o] Error 1
make[1]: Leaving directory '/home/me/src/vg/deps/ssw/src'
Makefile:106: recipe for target 'lib/libssw.a' failed
make: *** [lib/libssw.a] Error 2
```

I'm not sure if the issue is with SSE instructions, but this is a laptop I bought less than 2 years ago, not an ancient server, so that doesn't seem likely.

Are the README instructions out of date? Maybe there's a missing dependency?

--- a/README.md
+++ b/README.md
@@ -30,6 +30,8 @@ At present, you will need GCC version 4.9 or greater to compile vg. (Check your 
 
 Other libraries may be required. Please report any build difficulties.
 
+Note that a 64-bit OS is required. Ubuntu 16.04 should work.
+
 Now, obtain the repo and its submodules:
 
     git clone --recursive https://github.com/vgteam/vg.git

